plugins/woocommerce/includes/class-wc-brands.php

```diff
@@ -183,13 +183,13 @@ public function post_type_link( $permalink, $post ) {
 		// Get the custom taxonomy terms in use by this post.
 		$terms = get_the_terms( $post->ID, 'product_brand' );
 
-		if ( empty( $terms ) ) {
-			// If no terms are assigned to this post, use a string instead (can't leave the placeholder there).
-			$product_brand = _x( 'uncategorized', 'slug', 'woocommerce' );
-		} else {
-			// Replace the placeholder rewrite tag with the first term's slug.
-			$first_term    = array_shift( $terms );
-			$product_brand = $first_term->slug;
+		$product_brand = _x( 'uncategorized', 'slug', 'woocommerce' );
```

Comment:
Please consider retaining the comment. The bit about the placeholder sounds like it has some value.

Reply:
Thanks. I added it back here: 2426d36911073f0c33086ad11616fb9757b77c94